src/constants.rs

```diff
@@ -122,6 +122,15 @@ pub const DC_CONTACT_ID_LAST_SPECIAL: usize = 9;
 
 pub const DC_CREATE_MVBOX: usize = 1;
 
+#[repr(i32)]
+#[derive(Debug, Display, Clone, Copy, PartialEq, Eq, FromPrimitive, ToPrimitive, ToSql, FromSql)]
+pub enum Delay {
+    Do_not_try_again = 0,
```

Comment:
please use rusty name conventions for these `DoNotTryAgain`

Reply:
done